src/base/settingsstorage.cpp

```diff
@@ -82,7 +82,8 @@ namespace
             { "AddNewTorrentDialog/SavePathHistory", "TorrentAdditionDlg/save_path_history" },
             { "AddNewTorrentDialog/Enabled", "Preferences/Downloads/NewAdditionDialog" },
             { "AddNewTorrentDialog/TopLevel", "Preferences/Downloads/NewAdditionDialogFront" },
-            { "ExecutionLog/Enabled", "Preferences/ExecutionLog/enabled" }
+            { "ExecutionLog/Enabled", "Preferences/ExecutionLog/enabled" },
+            { "MainWindow/Notifications/Enabled", "Preferences/General/ProgramNotification" }
```

Comment:
Want to add that the subject of that mini-discussion was: "Shall settings key names follow component names, class names or option names?"